### PR TITLE
Updated newslist content to be more accurate to current events

### PR DESCRIPTION
**Pull request by Cascades**
Removed non-running events and updated hackpub text to show location and typical attenders

### CODE REVIEW

File: templates/newslist.html [hunk 1]
```diff
@@ -48,31 +48,16 @@ <h2>CoffeeScript</h2>
...
-
 <div class="infobox alternate">
   <h2>HackPub</h2>
-  <p>Another tradition with a long and glorious history, the humble HackPub, in which we visit one of York's many fine pubs and converse late into the night about high culture, politics, and how old jet engines work over a pint (of something non-alcoholic, if you so desire).</p>
-</div>
-
-<div class="infobox">
-  <h2>HackTogether</h2>
-  <p>A new addition to our roster, gathering in a computer lab together to work on projects either individually or together, and ordering pizza. Being alone together can be a lot more fun than being alone by yourself.</p>
+  <p>Another tradition with a long and glorious history, the humble HackPub, in which we visit the Rook and Gaskill at Walmgate and converse late into the night with members <b>past<\b> and present about high culture, politics, and how old jet engines work over a pint (of something non-alcoholic, if you so desire).</p>
```
Comment (LordAro) [inline templates/newslist.html]: "at Walmgate" - "at", and it's on Lawrence Street

Also, backslash

Comment (LordAro) [reply templates/newslist.html]: And it's not *entirely* set that it's at R&G

Comment (LukeMoll) [reply templates/newslist.html]: What should "at Walmgate" be changed to?

Comment (Cascades) [reply templates/newslist.html]: @LordAro #58 tried to fix, *new* committee today decided that there needed to be a location and also a little more emphasis on the amount of alumni, postgrads, (non-freshers basically) there :P

**Pull request merged**